cmd/translate/main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/spf13/cobra"
```

Comment:
Use hide whitespace, to ease the review :)
<img width="249" alt="image" src="https://github.com/translate-agent/translate/assets/69954124/8b4bda5a-eec6-4657-8cb5-f3660bbe3a16">